tests/unit/rest_api_httpd.cc

```diff
@@ -39,16 +39,25 @@ namespace bpo = boost::program_options;
 
 using namespace seastar;
 using namespace httpd;
+using namespace api_json::ns_hello_world;
 
 void set_routes(routes& r) {
-    api_json::hello_world.set(r, [] (const_req req) {
+    api_json::hello_world.set(r, [] (const_req req) -> json::json_return_type {
         api_json::my_object obj;
         obj.var1 = req.param.at("var1");
         obj.var2 = req.param.at("var2");
-        api_json::ns_hello_world::query_enum v = api_json::ns_hello_world::str2query_enum(req.query_parameters.at("query_enum"));
+        query_enum v = str2query_enum(req.query_parameters.at("query_enum"));
         // This demonstrate enum conversion
         obj.enum_var = v;
-        return obj;
+        stream_enum is_streaming =str2stream_enum(req.query_parameters.at("stream_enum"));
```

Comment:
might want to add spaces around `=`.

Reply:
Eagle eyes :eagle: !

Fixed in https://github.com/scylladb/seastar/commit/9f222060c1e51bc2c96cc8b2a49023b140d00eae.